src/components/GoalsPage/GoalsPage.tsx

```diff
@@ -217,7 +217,7 @@ export const GoalsPage = ({ user, ssrTime, defaultPresetFallback }: ExternalPage
                             tags={g.tags}
                             priority={g.priority}
                             comments={g._count?.comments}
-                            estimate={g._estimate?.length ? g._estimate[g._estimate.length - 1] : undefined}
+                            estimate={g.estimate?.length ? g.estimate[0].estimate : undefined}
```

Comment:
Why this is not `goal._lastEstimate` instead?

Reply:
changed to _lastEstimate